libs/vm-modules/include/vm_modules/ml/model/model_estimator.hpp

```diff
@@ -218,6 +218,9 @@ class ModelEstimator
   static ChargeAmount MaximumCharge(std::string const &log_msg = "");
 
   static ChargeAmount ToChargeAmount(fixed_point::fp64_t const &val);
+
+  static ChargeAmount constexpr EXPERIMENTAL_LARGE_COST  = std::numeric_limits<uint32_t>::max();
```

Comment:
I would suggest to use int32_T MAX instead of uint 'cause we actually have a forward_pass_cost with signed 32 bit mantissa

Reply:
Agreed

Reply:
Removed constatns completely. unnecessary.